README.md

```diff
@@ -109,11 +109,16 @@ Otherwise :violin: kindly ask someone from the dev team to help troubleshoot.
 
 ### Optional configuration options
 
-| Environment variable          | Default   | Description
-| ----------------------------- | --------- | -----------
-| ENABLE_KAFKA                  | false     | Feature flag to send kafka messages when a collection is published
-| KAFKA_ADDR                    | localhost:9092 | Comma seperated list of kafka brokers
-| KAFKA_CONTENT_PUBLISHED_TOPIC | content-published | Kafka topic to send content-published messages to
+| Environment variable                  | Default           | Description
+| --------------------------------------| ----------------  | -----------
+| ENABLE_KAFKA                          | false             | Feature flag to send kafka messages when a collection is published
+| KAFKA_ADDR                            | localhost:9092    | Comma seperated list of kafka brokers
+| KAFKA_CONTENT_PUBLISHED_TOPIC         | content-published | Kafka topic to send content-published messages to
+| KAFKA_SEC_PROTO                       | _unset_           | if set to "TLS", kafka connections will use TLS
+| KAFKA_SEC_CLIENT_KEY                  | _unset_           | if using TLS (see above), this is the path to the keystore (optional, used for client auth) - only used if `KAFKA_SEC_CLIENT_KEY_P12` is unset
+| KAFKA_SEC_CLIENT_KEY_P12              | _unset_           | if using TLS (see above), this is a base64-encoded PKCS12 keystore (optional, used for client auth)
```

Comment:
I'm just wondering if the names of these environment variables are misleading. These are a path-to or a B64 encoded keystore respectively rather than individual keys. Maybe something like the following would be more appropriate? (Happy for alternative suggestions.
```suggestion
| KAFKA_SEC_CLIENT_KEYSTORE_PATH                  | _unset_           | if using TLS (see above), this is the path to a disk-based keystore (optional, used for client auth) - only used if `KAFKA_SEC_CLIENT_KEYSTORE` is unset
| KAFKA_SEC_CLIENT_KEYSTORE              | _unset_           | if using TLS (see above), this is a base64-encoded PKCS12 keystore as a sinlge string (optional, used for client auth)
```

Reply:
use of KAFKA_SEC_CLIENT_KEY removed. Documentation update in another PR with more readMe updates